_docs/_latest/develop/index.md

```diff
@@ -23,11 +23,14 @@ GitHub.
 
 Get set up and started with development.
 
+**[Install, Buiild, and Run Forseti Website]({% link _docs/latest/develop/website.md %})**
```

Comment:
s/Buiild/Build/

Reply:
Also be sure that the order of the index.md also matches the ordering of the left sidebar.